freud/density.pyx

```diff
@@ -402,6 +407,11 @@ cdef class ComplexCF:
     def __str__(self):
         return repr(self)
 
+    def _repr_png_(self):
+        return freud.common.line_plot(self.R, np.real(self.RDF),
+                                      title="Complex CF",
+                                      xlabel="r", ylabel="C(r)")
```

Comment:
Try this: `ylabel="\Re(C(r))"` - we want to indicate it's the Real part.

Reply:
Do people usually use this \Re for real part? I feel like ones I have seen just put normal text Re.